lang/docs/src/doc.rs

```diff
@@ -0,0 +1,126 @@
+use opener;
+use std::fs;
```

Comment:
We use the following import order as a convention:

1. `std`
2. 3dparty crates (e.g. `opener`)
3. workspace crates
4. this crate

Hence, reformat as:

```rust
use std::fs;
use std::io;
use std::io::prelude::*;
use std::path::Path;
use std::path::PathBuf;

use opener;

use driver::paths::CSS_PATH;
use driver::Database;
use printer::{Print, PrintCfg};
```

In this case, since you are referring to the `opener` crate directly, you can also probably remove the `use opener` statement entirely.